pkg/synopsysctl/cmd_stop.go

```diff
@@ -105,9 +110,12 @@ var stopBlackDuckCmd = &cobra.Command{
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 		errors := []error{}
-		for _, bdArg := range args {
-			blackDuckName, blackDuckNamespace, crdNamespace, _, err := getInstanceInfo(false, util.BlackDuckCRDName, util.BlackDuckName, namespace, bdArg)
+		for _, blackDuckName := range args {
+			blackDuckNamespace, crdNamespace, scope, err := getInstanceInfo(false, util.BlackDuckCRDName, util.BlackDuckName, namespace, blackDuckName)
 			if err != nil {
+				if len(blackDuckNamespace) == 0 && scope == apiextensions.ClusterScoped {
+					err = fmt.Errorf("%s %s doesn't appear to be running: %v", util.BlackDuckName, blackDuckName, err)
```

Comment:
getInstanceInfo method will give the namespace that need to be created, it won't check if the namespace is existed or not. Either we need to check whether the namespace exist or need to check the state of the CR is running